### DISCUSSION TITLE
Google Beta updated to new version but Tabs Outliner tree was gone and discovered Link-Map

### DISCUSSION THREAD

**Discussion by Almighty188** [GarinZ/link-map#48] [Q&A]
Yesterday, Google Chrome released a new Beta version and when it relaunched, it cleaned out the Tabs Outliner tree which was how I discovered Link-Map.  Is there anyway that Link-Map can regenerate that tree which I have saved and unloaded.  I forgot to do a periodic backup but I can recover the deleted files.

**Comment by GarinZ** [comment]
Hi @Almighty188,
If there is a exported json file(`tree-exported-xx.tree`) of Tabs Outliner. It could be regenerated in both Tabs Outliner or Link Map.
Or If you could providing me any file of Tabs Outliner(e.g. backup html page) by email, I could try to recover it personally.

**Reply by Almighty188** [reply]
I don't have any exported files from Tabs Outliner or else I wouldn't  be in he crisis I am now.  The only thing I have is the IndexedDB\chrome-extension_eggkanocgddhmamlbiijnphhppkpkmkl_0.indexeddb.blob and IndexedDB\chrome-extension_eggkanocgddhmamlbiijnphhppkpkmkl_0.indexeddb.leveldb right after the issue and the files it deleted from one day which was two launches of Google Chrome before the issue.  I had e-mailed the files in the two directories 2 days ago and I believe if there was any hope of recovering it, you might be one of two persons, the other being the author of Tabs Outliner who even knows how this type of tool works with the high in depth knowledge.
